src/Cardano/CoinSelection.hs

```diff
@@ -57,22 +68,24 @@ data CoinSelection = CoinSelection
 --
 instance Semigroup CoinSelection where
     a <> b = CoinSelection
-        { inputs = inputs a <> inputs b
+        { inputs  = inputs  a <> inputs  b
         , outputs = outputs a <> outputs b
-        , change = change a <> change b
+        , change  = change  a <> change  b
         }
 
 instance Monoid CoinSelection where
     mempty = CoinSelection [] [] []
 
 instance Buildable CoinSelection where
-    build (CoinSelection inps outs chngs) = mempty
-        <> nameF "inputs" (blockListF' "-" inpsF inps)
-        <> nameF "outputs" (blockListF outs)
-        <> nameF "change" (listF chngs)
+    build cs = mempty
+        <> nameF "inputs"  (blockListF' "-" inpsF $ inputs  cs)
+        <> nameF "outputs" (blockListF            $ outputs cs)
+        <> nameF "change"  (listF                 $ change  cs)
```

Comment:
this is compliant with our coding standards?

Reply:
@paweljakubas wrote:

> this is compliant with our coding standards?

You're right -- it is not compliant.

I was under the impression that we had recently relaxed this standard, but it turns out that we haven't!

Therefore, I have removed this change (and replaced it with a simplification).

Thanks for pointing this out!